packages/algob/test/lib/tx.ts

```diff
@@ -555,4 +555,25 @@ describe("Delete ASA and SSC transaction flow(with functions and executeTransact
 
     await executeTransaction(deployer, execParam);
   });
+
+  it("should pass if user tries delete app that doesn't exist in checkpoint", async () => {
+    const txGroup: ExecParams[] = [
+      {
+        type: types.TransactionType.DestroyAsset,
+        sign: types.SignType.SecretKey,
+        fromAccount: bobAcc,
+        payFlags: {},
+        assetID: 123
+      },
+      {
+        type: types.TransactionType.DeleteSSC,
+        sign: types.SignType.SecretKey,
+        fromAccount: bobAcc,
+        payFlags: {},
+        appID: 12213
+      }
+    ];
+
+    await executeTransaction(deployer, txGroup);
```

Comment:
how about using `assert.notthrows(func, ERRORS.GENERAL.APP_DELETED)` here

Reply:
OK

Reply:
but that means it could throw another error ?

Reply:
if that's the case, `await exec.....` is better